addons/interaction/functions/fnc_manageAce3Interactions.sqf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Receives a status update and sets ACE3 dragging, carrying and cargo accordingly.
+ *
+ * Arguments:
+ * 0: Equipment <OBJECT>
+ * 1: Condition <STRING>
+ * 2: Status <BOOLEAN>
+ *
+ * Returns:
+ * none
+ *
+ * Example:
+ * [_computer, "powerConnected", true] call AE3_interaction_fnc_manageAce3Interactions
+ */
+
+params ["_target", "_condition", "_status"];
+
+// ace_dragging_fnc_setCarryable and ace_dragging_fnc_setDraggable need to be executed on all machines with JIP
+// ace_cargo_fnc_setSize can be executed locally because it handles JIP and remoteExecCall on it's own
+
+//this function only runs once per triggered interaction by player on players machine
```

Comment:
Why not run this server side only? All effects of this function are made globally anyway

Reply:
Ok, I will try to change that. But I think the only benefit is that this chunk of code (that already only runs once) is then executed on the server instead of the client. I think I will simply change that by remote executing this function on the server.

```sqf
// doing this
remoteExecCall 
[_computer, "inUse", false] remoteExecCall ["AE3_interaction_fnc_manageAce3Interactions", 2];

// instead of this
[_computer, "inUse", false] call AE3_interaction_fnc_manageAce3Interactions;
```